apps/keycloak/vars/all/main.yml

```diff
@@ -13,7 +13,8 @@ keycloak_database_secret_name: "keycloak-database-{{ keycloak_vault_checksum | d
 
 # -- keycloak
 keycloak_image_name: "quay.io/keycloak/keycloak"
-keycloak_image_tag: "latest"
+# FIXME
```

Comment:
Is it a FIXME because with the latest, the tray is broken ?

Reply:
Yes!